README.md

```diff
@@ -42,6 +42,12 @@ to find out more about the origin story and design philosophy of Envoy.
 Please see [this](https://groups.google.com/forum/#!topic/envoy-announce/l9zjYsnS3TY) email thread 
 for information on email list usage.
 
+## Community Meeting
+
+The Envoy team meets every other Tuesday at 8am PT. The public google calendar is here: https://goo.gl/PkDijT
```

Comment:
8am or 9am?  When viewing the public calendar, it says 11am but doesn't specify the timezone.  When I copy it to my calendar, it said 9am PT.